mappings/net/minecraft/world/loot/function/SetTagLootFunction.mapping

```diff
@@ -1,3 +1,10 @@
 CLASS cpv net/minecraft/world/loot/function/SetTagLootFunction
```

Comment:
All other functions are based on the name of the function, like it is called in the vanilla geme (`set_contents` becomes `SetContentsLootFunction`)
So why does `set_nbt` become `SetTagLootFunction`? Is it okay to change that?

Reply:
Feel free to change that.